<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sell</class>
 <widget class="QDialog" name="Sell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sell</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lie_itemname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>67</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上架商品</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>底价(元)：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lie_price">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>商品详情：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txe_detail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_upLoad">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
